Sources/PackageCollectionsModel/Formats/v1.md

```diff
@@ -0,0 +1,237 @@
+# Package Collection
```

Comment:
Moving the format spec to this repo so that it's closer to the source. Previously it lives in the generator repo.

Reply:
That's a good idea.  Does the generator repo reference this one for the spec?

Reply:
Yep I am updating the doc there to link to this.